### PR TITLE
Log-verbosity

### PR DESCRIPTION
**Pull request by Bart6114**
Package uses logging functionalty but introduces a "verbose" concept where loglevel INFO suddendly becomes more verbose. This is a small PR that switches log level level to DEBUG when `--verbose` flag is passed by the user.

### CODE REVIEW

File: databooks/common.py [hunk 1]
```diff
@@ -21,6 +24,14 @@ def get_logger(name: str, level: str = "INFO") -> logging.Logger:
     return logging.getLogger(name)
 
 
+def set_verbose(logger: logging.Logger) -> None:
```
Comment (murilo-cunha) [inline databooks/common.py:27]: Could you add a oneliner docstring here?

File: databooks/common.py [hunk 2]
```diff
@@ -21,6 +24,14 @@ def get_logger(name: str, level: str = "INFO") -> logging.Logger:
...
+    verbose_level = logging.DEBUG
+    logger.setLevel(verbose_level)
+    logger.debug(
+        f"verbose mode: setting log level to {logging.getLevelName(verbose_level)}"
```
Comment (murilo-cunha) [inline databooks/common.py:31]: My OCD kicks in: could you capitalize the first letter? 😅

**Comment by murilo-cunha** [comment]
Thanks for the PR. I like the changes! I left some minor pedantic requests 😊.

**Review by murilo-cunha — CHANGES_REQUESTED** [review]
Looks good! Just left minor things on `databooks/common.py`

File: databooks/common.py (hunk 1 above)
Comment (Bart6114) [reply databooks/common.py:27]: Aye captain, BTW if we'd use flake8-docstrings the CI would've blocked it. I added #5 for your consideration.

File: databooks/common.py (hunk 2 above)
Comment (Bart6114) [reply databooks/common.py:31]: Affirmative. BTW, this comes from go, where people start complaining when I use a capitals to start log/error messages 😅 .

**Review by murilo-cunha — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**